pkg/vulnerability/vulnerability.go

```diff
@@ -46,6 +48,12 @@ var SuperSet = wire.NewSet(
 	NewClient,
 )
 
+// Show warning if we use severity from another vendor
+// cf. https://github.com/aquasecurity/trivy/issues/6714
+var onceWarn = sync.OnceFunc(func() {
+	log.Warn("For some vulnerabilities, severities from other vendors were used because their data sources and NVD don't contain severities. You can view severity list from vendors in `json` format in `VendorSeverity` field.")
```

Comment:
After reading the message, I feel like it's still unclear for users. We may want to update [the document](https://aquasecurity.github.io/trivy/v0.52/docs/scanner/vulnerability/#severity-selection) and add a link in the warning message, like "For details, read https://aquasecurity.github.io/trivy/latest/docs/scanner/vulnerability/#severity-selection." What do you think?

Reply:
I have a couple of ideas that we can add. I will update docs and write to you.

Reply:
https://github.com/aquasecurity/trivy/pull/6726/commits/ec412a332a3c30572ff153bc622247643a86b1ca
wdyt?